doc/source/specification.rst

```diff
@@ -23,32 +23,65 @@ does not make any sense in any of the other soma formats.
 Soma single point SOMA_SINGLE_POINT
 ***********************************
 
-If a soma is made of only one point it will be treated as a "single point soma" regardless of the
-morphology file format. The single point soma is represented as a sphere whose coordinate and
-radius are given by the soma point.
+If a soma is made of only one point it will be treated as a "single point soma" for H5 and ASC formats.
+The single point soma is represented as a sphere whose coordinate and radius are given by the soma point.
+SWC interprets such soma differently.
 
 Morphology file dependent soma type:
 ************************************
 
 SWC
 ^^^
 
-`SOMA_NEUROMORPHO_THREE_POINT_CYLINDERS <https://github.com/BlueBrain/MorphIO/blob/a60b52dfe403ef289455ee2221c1b4fce6418978/src/plugin/morphologySWC.cpp#L206>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-SWC files with a soma made of 3 points will have a soma of type 3-points soma. A 3-points soma is
+Soma single point
+~~~~~~~~~~~~~~~~~
+.. image:: images/swc/single.svg
+   :scale: 100 %
+   :alt: soma single point
+
+A single point sphere soma is interpreted as a cylinder of equivalent area.
```

Comment:
I think we should be more precise about the "point". This is actually not a point but a sphere/circle --> point + a diameter. 
The "point" definition differs from the single point and the two point soma.

In the single point this is a sphere. In the 2 point soma these are 2 circles.
> two points as centers of the cylinder's bases

Reply:
Am I correct ?

Reply:
I will adjust. In my opinion the whole docs require some serious cleaning. I created a ticket for that #277.

Reply:
Feel free to fix my definitions. I find it hard to adjust only SWC parts without affecting H5 and ASC. In my opinion we need a good general definition, that will be further concretized for each format. Currently MorphIO does not give general definition.

Reply:
I think this should be done in a much bigger review about docs. But this takes time and this is not the priority yet. Your current definition is great atm.